packages/video_player/ios/Classes/VideoPlayerPlugin.m

```diff
@@ -198,14 +290,42 @@ - (void)updatePlayingState {
   _displayLink.paused = !_isPlaying;
 }
 
+static inline CGFloat radiansToDegrees(CGFloat radians) {
```

Comment:
Perhaps use the solution from https://stackoverflow.com/a/29352646

Reply:
I've replaced `CGFloat degrees = radians * 180 / M_PI;` with  `GLKMathRadiansToDegrees`. However, I still need to keep the code below to bound the rotation degrees between [0 and 360[:
```OBJECTIVE-C
 if (degrees < 0) {
    // Convert -90 to 270 and -180 to 180
    return degrees + 360;
  }
  // Output degrees in between [0, 360[
  return degrees;
```